nucypher/policy/models.py

```diff
@@ -468,7 +467,55 @@ def __len__(self):
         return len(self.destinations)
 
 
-class WorkOrder(object):
+class RevocationNotice:
+    splitter = BytestringSplitter((bytes, 7),
+                                  (bytes, constants.NOTICE_LENGTH(32)),
+                                  Signature)
+    def __init__(self, node_id, arrangement_id, signature=None):
+        self.node_id = node_id
+        self.arrangement_id = arrangement_id
+        self.notice = b"REVOKE-" + arrangement_id
```

Comment:
This pattern really gets my :goat:.  If the whole class is the `RevocationNotice`, then what the heck is this?

Which thing is *the* notice?